app/assets/javascripts/admin/addon/components/admin-watched-word.js

```diff
@@ -16,18 +16,20 @@ export default Component.extend({
     return replacement.split(",");
   },
 
-  click() {
-    this.word
-      .destroy()
-      .then(() => {
-        this.action(this.word);
-      })
-      .catch((e) => {
-        bootbox.alert(
-          I18n.t("generic_error_with_reason", {
-            error: `http: ${e.status} - ${e.body}`,
-          })
-        );
-      });
+  actions: {
+    deleteRecord() {
```

Comment:
```suggestion
    deleteWord() {
```

How about the above? `record` seems like a very generic term to use.

Reply:
Can we use the `@action` decorator here?

Reply:
Good suggestion, done! 👍🏼